compute_tools/src/compute.rs

```diff
@@ -711,7 +711,7 @@ impl ComputeNode {
         let mut file = std::fs::File::create(conf_path)?;
         writeln!(file, "shared_buffers=65536")?;
         writeln!(file, "port=51055")?; // Nobody should be connecting
-        writeln!(file, "shared_preload_libraries = 'neon'")?;
+        writeln!(file, "shared_preload_libraries = 'neon,pg_partman_bgw'")?;
```

Comment:
This should be done in `neondatabase/cloud` repo (in `settings.go` file we have a list of `shared_preload_libraries`)

Reply:
Got it, removed this